go.mod

```diff
@@ -143,3 +142,5 @@ retract (
 	v1.0.1 // This version is used only to publish retraction of v1.0.1.
 	v1.0.0 // We reverted to v0.… version numbers; the v1.0.0 tag was actually deleted.
 )
+
+replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.1-0.20220617142545-8b9452f75cbc
```

Comment:
Given the lack of notice / description, I assume this is unwanted.

Reply:
(I don’t have an opinion on the _substance_ of the `runc` dependency and/or the choice to use `replace` here; merely noting that the version was recently intentionally moved on `main` and this seems to change that decision.)

Reply:
Yes this should not be there.  We are only deciding to ignore updates in podman, because of bloat.

Reply:
it was a mistake during rebasing, this PR should not affect the runc dependency.

Updated now